src/nrc/api/tests/test_subscriptions.py

```diff
@@ -214,10 +214,11 @@ def test_partial_update_subscriptions(self):
             )
             response = self.client.patch(subscription_update_url, data)
 
-        self.assertEqual(
-            response.status_code, status.HTTP_403_FORBIDDEN
-        )  # scope check is done before returning 405
-        self.assertEqual(subscription.types, None)
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+        subscription.refresh_from_db()
+
+        self.assertEqual(subscription.types, data["types"])
```

Comment:
Dit stukje is waarschijnlijk bedoeld voor de wijziging die je in een andere PR gaat doen

Reply:
Klopt maar met de nieuwe scopes faalt deze test nu (want je mag wel degelijk een PATCH operatie uitvoeren als je `heeft_alle_autorisaties = True` hebt staan). Vandaar dat hij hier ook al gewijzigd moest worden.

Reply:
Ik zie het nu, de scope was nog niet aanwezig voor `patch` maar nu dat hij er is en de operatie er ook al inzat (maar dus nog een scope miste) werkt de operatie in zijn geheel in een keer